spark-connector/src/test/java/com/datastrato/gravitino/spark/connector/TestSparkTypeConverter.java

```diff
@@ -59,6 +63,60 @@ void init() {
     gravitinoToSparkTypeMapper.put(DateType.get(), DataTypes.DateType);
     gravitinoToSparkTypeMapper.put(TimestampType.withTimeZone(), DataTypes.TimestampType);
     gravitinoToSparkTypeMapper.put(TimestampType.withoutTimeZone(), DataTypes.TimestampNTZType);
+    gravitinoToSparkTypeMapper.put(
+        ListType.of(IntegerType.get(), true), DataTypes.createArrayType(DataTypes.IntegerType));
+    gravitinoToSparkTypeMapper.put(
+        MapType.of(IntegerType.get(), StringType.get(), true),
+        DataTypes.createMapType(DataTypes.IntegerType, DataTypes.StringType));
+    gravitinoToSparkTypeMapper.put(createGravitinoStructType(), createSparkStructType());
```

Comment:
How about this? create static method for gravitino and spark to build their structType. cc @FANNG1

Reply:
ok, please provide some description to the two functions, and move private functions after public functions.

Reply:
done.